__init__.py

```diff
@@ -162,6 +162,13 @@ def _handle_client_exception(self, callback, *args, **kwargs):
         return False
 
     # Intent handlers
+    @intent_handler('camera.intent')
+    def handle_camera_intent(self, message):
+        """Handle camera intent."""
+        message.data["Entity"] = message.data.get("entity")
+        message.data["Action"] = "on"
```

Comment:
Does this get used? 

I'm guessing the intention is to allow the `_handle_camera_actions()` method to be extended in the future to handle different types of actions?

If there's only one Action type at the moment, maybe we rename the method to specify what it does now eg `_show_camera_image()`. Then in the future we can call it from a more generic camera actions handler?